### DISCUSSION TITLE
[Ctrl+F5] VSCode - Não foi possível localizar a tarefa 'build'

### DISCUSSION THREAD

**Discussion by marcelobarbieri** [balta-io/7194#5] [Perguntas e Respostas] [answered]
Bom dia.

Ao executar Ctrl + F5 está sendo exibida a mensagem "Não foi possível localizar a tarefa 'build'".
Por gentileza, poderia me auxiliar?

Desde já obrigado.

`.vscode/lauch.json
`{
    // Use o IntelliSense para saber mais sobre os atributos possíveis.
    // Focalizar para exibir as descrições dos atributos existentes.
    // Para obter mais informações, acesse: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": ".NET Core Launch (console)",
            "type": "coreclr",
            "request": "launch",
            "WARNING01": "*********************************************************************************",
            "WARNING02": "The C# extension was unable to automatically decode projects in the current",
            "WARNING03": "workspace to create a runnable launch.json file. A template launch.json file has",
            "WARNING04": "been created as a placeholder.",
            "WARNING05": "",
            "WARNING06": "If OmniSharp is currently unable to load your project, you can attempt to resolve",
            "WARNING07": "this by restoring any missing project dependencies (example: run 'dotnet restore')",
            "WARNING08": "and by fixing any reported errors from building the projects in your workspace.",
            "WARNING09": "If this allows OmniSharp to now load your project then --",
            "WARNING10": "  * Delete this file",
            "WARNING11": "  * Open the Visual Studio Code command palette (View->Command Palette)",
            "WARNING12": "  * run the command: '.NET: Generate Assets for Build and Debug'.",
            "WARNING13": "",
            "WARNING14": "If your project requires a more complex launch configuration, you may wish to delete",
            "WARNING15": "this configuration and pick a different template using the 'Add Configuration...'",
            "WARNING16": "button at the bottom of this file.",
            "WARNING17": "*********************************************************************************",
            "preLaunchTask": "build",
            "program": "${workspaceFolder}/bin/Debug/<insert-target-framework-here>/<insert-project-name-here>.dll",
            "args": [],
            "cwd": "${workspaceFolder}",
            "console": "internalConsole",
            "stopAtEntry": false
        },
        {
            "name": ".NET Core Attach",
            "type": "coreclr",
            "request": "attach"
        }
    ]
}``

![image](https://user-images.githubusercontent.com/86416342/125773933-207c35e5-6be1-4ba8-8b7f-a11790c92d4b.png)

![image](https://user-images.githubusercontent.com/86416342/125773989-3a8e12f8-72f4-4e7f-8c0e-d0ff12fd6bd2.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @mdpbarbieri, isto ocorreu por que você ignorou a mensagem de criar os assets para depurar aplicações .NET Core no start do projeto... mas não tem problema, dá pra gerar assim:

Crie um novo `launch.json`
![image](https://user-images.githubusercontent.com/965305/125785344-d1b3680b-fd01-4ac5-b263-836413bafd58.png)

Selecione a opção `Run and Debug` e `.NET Core`
![image](https://user-images.githubusercontent.com/965305/125785397-33517cc3-4178-44cf-97ab-62499a053c07.png)

Rode a aplicação
![image](https://user-images.githubusercontent.com/965305/125785433-4f7cf8f4-0988-47f1-b3ad-b7ed7644ad49.png)


É para funcionar :)

**Reply by marcelobarbieri** [reply]
Balta, muito obrigado. Ao criar o projeto webapi não lembro de ter aparecido a mensagem. Geralmente clico em Yes para criar os assets. Estou utilizando o dotnet na versão 6 preview. Foi resolvido seguindo as suas instruções.
![image](https://user-images.githubusercontent.com/86416342/125870628-ab7376f3-d088-47eb-bfba-09d278ecddb9.png)

Muito obrigado pelo suporte!
